go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/DataDog/zstd v1.4.0 // indirect
 	github.com/alecthomas/participle v0.2.1
 	github.com/aliyun/aliyun-oss-go-sdk v0.0.0-20190307165228-86c17b95fcd5
+	github.com/aws/aws-sdk-go v1.20.20
```

Comment:
Why is this suddenly needed?

Reply:
Again auto-added by `go mod` due to `go mod tidy`